#inputs
bool setLinActuatorState
---  
#outputs
#bool isLinActuatorOn